package.json

```diff
@@ -12,12 +12,14 @@
     "pretest": "lerna run pretest",
     "test": "node scripts/test.js",
     "posttest": "lerna run posttest",
-    "test:watch": "node scripts/test.js --watch",
+    "test:watch": "run-script-os",
+    "test:watch:darwin:linux": "sleep 10 && yarn test:watch:default",
+    "test:watch:default": "node scripts/test.js --watch",
     "lint": "tsc -b && yarn prettier:check && eslint --ext .js,.jsx,.ts,.tsx,.graphql .",
-    "lint:fix": "eslint --ext .js,.jsx,.ts,.tsx,.graphql . --fix; prettier --ignore-path .eslintignore --write '**/*.{js,jsx,ts,tsx,graphql,md}'",
-    "prettier:check": "prettier --ignore-path .eslintignore --check '**/*.{js,jsx,ts,tsx,graphql,md}'",
+    "lint:fix": "eslint --ext .js,.jsx,.ts,.tsx,.graphql . --fix && prettier --ignore-path .eslintignore --write \"**/*.{js,jsx,ts,tsx,graphql,md}\"",
```

Comment:
If we're going with an `&&` here, we should change the order so that prettier always runs (as it does currently) since it's rare for prettier to fail and I definitely want `lint:fix` to format my code even if there's ESLint errors. I have concerns that some ESLint rules might attempt formatting which prettier would undo, so if this _does_ cause reformatting (it shouldn't) we should track down the rules causing this and disable them. Or we could `prettier && eslint && prettier` but that seems inefficient.

Reply:
I changed it to `&&` because `;` is a single `&` on Windows. Therefore I replaced it with `run-script-os` for changing the command to a Windows specific version.